README.adoc

```diff
@@ -3216,6 +3216,42 @@ def some_method(&)
 end
 ----
 
+=== Super Forwarding
+
+Use `super` without arguments when the enclosing methods arguments are the same.
+
+When super is called without arguments and parentheses Ruby will automatically forward the arguments taken by the method.
+
+[source,ruby]
+----
+# bad
+def some_method(*args, **kwargs)
+  super(*args, **kwargs)
+end
+
+# good - implicitly passing all arguments
+def method(*args, **kwargs)
+  super
+end
+
+# good - forwarding a subset of the arguments
+def method(*args, **kwargs)
+  super(*args)
+end
+
+# good - calling super with different arguments
+def method(*args, **kwargs)
+  super("foo", *args, **kwards)
```

Comment:
I haven’t tried, but would `super(“foo”, …)` syntax work?

Reply:
This does work in Ruby 3.1 (I think). `...` got introduced and one or two versions later this was made valid syntax